src/css/controls/imports/rightclick.less

```diff
@@ -14,57 +13,52 @@
         display: block;
     }
 
-    ul {
+    .jw-rightclick-list {
+        border-radius: 1px;
         list-style: none;
-        font-weight: bold;
-        border-radius: 0.15em;
         margin: 0;
-        border: 1px solid @rightclick-border;
         padding: 0;
 
-        li {
+        .jw-rightclick-item {
             background-color: @rightclick-bg;
             border-bottom: 1px solid @rightclick-border;
             margin: 0;
 
             .jw-rightclick-logo {
-                font-size: 2em;
-                color: #ff0147;
+                color: @white;
+                display: inline-flex;
+                padding: 0 10px 0 0;
                 vertical-align: middle;
-
-                padding-right: 0.3em;
-                margin-right: 0.3em;
-                border-right: 1px solid @rightclick-border;
             }
 
-            a {
+            .jw-rightclick-link {
+                border: none;
                 color: @rightclick-primary-text;
-                text-decoration: none;
-
-                // padding comes from the anchor, so that the whole item is clickable
-                padding: 1em;
                 display: block;
-                font-size: (11 / 16em);
+                font-size: 11px;
```

Comment:
Should this be `0.6875em`?

Reply:
IE and Edge chop values down to the nearest hundredth by default